src/EventStore.Core/Caching/MemSizer.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using EventStore.Common.Utils;
+
+namespace EventStore.Core.Caching {
+	public class MemSizer {
+		public const int ObjectHeaderSize = 16;
+		public const int ArraySize = 24;
+		public const int StringSize = 20;
+
+		public static int SizeOf(string s)
+		{
+			if (s.Length == 0)
```

Comment:
consider checking for null string

Reply:
fixed in: 57a260ecf1f37450ebc517760d4b8ce7682e764b